L1Trigger/L1TMuonEndCap/src/bdt/Node.cc

```diff
@@ -2,13 +2,13 @@
 //                            Node.cxx                                  //
 // =====================================================================//
 // This is the object implementation of a node, which is the            //
```

Comment:
Leave file as is.

Reply:
Since there's no changes except spaces, it should be trivial to review. I'd really prefer to have no trailing spaces and tabs.